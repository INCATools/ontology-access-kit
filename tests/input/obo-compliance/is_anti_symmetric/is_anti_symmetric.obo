format-version: 1.4
ontology: is_anti_symmetric

[Typedef]
id: R:1
is_anti_symmetric: true

