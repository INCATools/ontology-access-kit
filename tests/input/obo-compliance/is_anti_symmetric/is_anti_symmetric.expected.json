{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/is_anti_symmetric.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/IAO_0000427",
      "lbl" : "antisymmetric property",
      "type" : "PROPERTY"
    }, {
      "id" : "http://purl.obolibrary.org/obo/R_1",
      "type" : "PROPERTY",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://purl.obolibrary.org/obo/IAO_0000427",
          "val" : "true"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ]
  } ]
}